UI/data/locale/en-US.ini

```
@@ -946,6 +946,7 @@ Basic.Settings.Output.ReplayBuffer.EstimateTooLarge="Warning: Estimated memory u
 Basic.Settings.Output.ReplayBuffer.EstimateUnknown="Cannot estimate memory usage. Please set maximum memory limit."
 Basic.Settings.Output.ReplayBuffer.Prefix="Replay Buffer Filename Prefix"
 Basic.Settings.Output.ReplayBuffer.Suffix="Suffix"
+Basic.Settings.Output.ReplayBuffer.UnavailableCustomFFmpeg="Replay Buffer is unavailable when using Custom Output (FFmpeg)."
```

Potentially for clarity. Up to Warchamp in the end.
```suggestion
Basic.Settings.Output.ReplayBuffer.UnavailableCustomFFmpeg="Replay Buffer is unavailable when Recording is set to Custom Output (FFmpeg)."
```

---

I agree with this